package.json

```diff
@@ -5,16 +5,18 @@
   "author": "bvaughn",
   "license": "MIT",
   "repository": "bvaughn/react-virtualized-v10",
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
+  "main": "dist/index.cjs.js",
```

Comment:
This is fine. Seems kind of like a subjective nit but I'm not really opposed to it.

Reply:
Yeah, it's just a subjective convention: cjs/esm/umd